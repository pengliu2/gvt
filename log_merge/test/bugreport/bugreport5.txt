=================================
== dumpstate: 2010-10-01 16:10:30
------ SYSTEM LOG
10-01 11:01:39.999 system
10-01 11:01:40.000 system
10-01 11:01:40.000 system
10-01 11:01:40.000 system
10-01 11:01:40.628 system
10-01 11:01:53.000 system
10-01 11:01:53.000 system
10-01 11:01:54.000 system
10-01 11:10:01.000 system
10-01 11:10:02.629 system
10-01 11:10:03.629 system
10-01 11:10:04.629 system
10-01 11:10:05.629 system
10-01 11:10:06.629 system
------ 

------ RADIO LOG
10-01 11:01:39.999 radio
10-01 11:01:39.999 radio
10-01 11:01:40.000 radio
10-01 11:01:40.000 radio
10-01 11:01:40.000 radio
10-01 11:01:40.001 radio
10-01 11:01:40.001 radio      
10-01 11:10:40.002 radio
10-01 11:10:02.628 radio
10-01 11:10:03.628 radio
10-01 11:10:04.628 radio
10-01 11:10:05.628 radio
10-01 11:10:06.629 radio
10-01 11:10:07.629 radio
------ 

------ KERNEL LOG
<7>[   13.886499] kernel
<6>[   13.886499] kernel
<6>[   13.886499] kernel
<6>[   13.967193] kernel
<6>[   20.000000] cpcap_rtc cpcap_rtc: setting system clock to 2010-10-01 16:01:30 UTC (1285948890)
<7>[   20.887237] kernel
<6>[   20.890533] kernel
<6>[   20.891815] kernel
<6>[   30.000000] kernel
<7>[   30.000000] kernel
<6>[   34.802825] kernel
<6>[   35.000000] suspend: exit suspend, ret = -16 (2010-10-01 16:01:50.000000000 UTC)
<6>[   35.910491] kernel
<6>[   35.910675] kernel
<4>[   37.999999] kernel
<4>[   38.000000] kernel
<6>[   38.000001] kernel
<4>[   35.913543] kernel
<6>[    1.000000] suspend: exit suspend, ret = -16 (2010-10-01 16:10:01.000000000 UTC)
<6>[    1.279693] kernel
<6>[    2.910491] kernel
<6[    3.910675] kernel
<4[    4.912536] kernel
<4[    5.912872] kernel
<6[    6.913360] kernel
<4[    7.913543] kernel
------ 
