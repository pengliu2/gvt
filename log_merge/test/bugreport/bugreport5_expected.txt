KERNEL LOG:<7>[   13.886499] kernel
KERNEL LOG:<6>[   13.886499] kernel
KERNEL LOG:<6>[   13.886499] kernel
KERNEL LOG:<6>[   13.967193] kernel
KERNEL LOG:<6>[   20.000000] cpcap_rtc cpcap_rtc: setting system clock to 2010-10-01 16:01:30 UTC (1285948890)
KERNEL LOG:<7>[   20.887237] kernel
KERNEL LOG:<6>[   20.890533] kernel
KERNEL LOG:<6>[   20.891815] kernel
SYSTEM LOG:10-01 11:01:39.999 system
RADIO LOG:10-01 11:01:39.999 radio
RADIO LOG:10-01 11:01:39.999 radio
KERNEL LOG:<6>[   30.000000] kernel
SYSTEM LOG:10-01 11:01:40.000 system
RADIO LOG:10-01 11:01:40.000 radio
KERNEL LOG:<7>[   30.000000] kernel
SYSTEM LOG:10-01 11:01:40.000 system
RADIO LOG:10-01 11:01:40.000 radio
SYSTEM LOG:10-01 11:01:40.000 system
RADIO LOG:10-01 11:01:40.000 radio
RADIO LOG:10-01 11:01:40.001 radio
RADIO LOG:10-01 11:01:40.001 radio      
SYSTEM LOG:10-01 11:01:40.628 system
KERNEL LOG:<6>[   34.802825] kernel
KERNEL LOG:<6>[   35.000000] suspend: exit suspend, ret = -16 (2010-10-01 16:01:50.000000000 UTC)
KERNEL LOG:<6>[   35.910491] kernel
KERNEL LOG:<6>[   35.910675] kernel
KERNEL LOG:<4>[   37.999999] kernel
SYSTEM LOG:10-01 11:01:53.000 system
KERNEL LOG:<4>[   38.000000] kernel
SYSTEM LOG:10-01 11:01:53.000 system
KERNEL LOG:<6>[   38.000001] kernel
KERNEL LOG:<4>[   35.913543] kernel
SYSTEM LOG:10-01 11:01:54.000 system
KERNEL LOG:<6>[    1.000000] suspend: exit suspend, ret = -16 (2010-10-01 16:10:01.000000000 UTC)
SYSTEM LOG:10-01 11:10:01.000 system
KERNEL LOG:<6>[    1.279693] kernel
SYSTEM LOG:10-01 11:10:02.629 system
KERNEL LOG:<6>[    2.910491] kernel
KERNEL LOG:<6[    3.910675] kernel
KERNEL LOG:<4[    4.912536] kernel
KERNEL LOG:<4[    5.912872] kernel
KERNEL LOG:<6[    6.913360] kernel
KERNEL LOG:<4[    7.913543] kernel
SYSTEM LOG:10-01 11:10:03.629 system
SYSTEM LOG:10-01 11:10:04.629 system
SYSTEM LOG:10-01 11:10:05.629 system
SYSTEM LOG:10-01 11:10:06.629 system
RADIO LOG:10-01 11:10:40.002 radio
RADIO LOG:10-01 11:10:02.628 radio
RADIO LOG:10-01 11:10:03.628 radio
RADIO LOG:10-01 11:10:04.628 radio
RADIO LOG:10-01 11:10:05.628 radio
RADIO LOG:10-01 11:10:06.629 radio
RADIO LOG:10-01 11:10:07.629 radio
